apps/crn-server/test/handlers/webhooks/fetch-by-code/fetch-by-code.test.ts

```diff
@@ -154,6 +154,22 @@ describe('Fetch-user-by-code handler', () => {
       expect(JSON.parse(result.body)).toEqual(getUserResponse());
     });
 
+    test('should return an empty string as the algolia key if the user is not onboarded', async () => {
```

Comment:
shouldn't it be null or undefined or something? although then i reckon you would need to assert that the key is present which imho is correct